### ISSUE TITLE
Cannot install gotk3 using either go get or go install

### ISSUE THREAD

**Issue by DThiebaud** [gotk3/gotk3#872]
Under Ubuntu Cinnamon, I get the followiing result:

dick@dick1:~$ go version
go version go1.18.1 linux/amd64
dick@dick1:~$ go get github.com/gotk3/gotk3/gtk
go: go.mod file not found in current directory or any parent directory.
	'go get' is no longer supported outside a module.
	To build and install a command, use 'go install' with a version,
	like 'go install example.com/cmd@latest'
	For more information, see https://golang.org/doc/go-get-install-deprecation
	or run 'go help get' or 'go help install'.
dick@dick1:~$ go install github.com/gotk3/gotk3/gtk@latest
package github.com/gotk3/gotk3/gtk is not a main package

Under Windows 10 I get the same result:

C:\Windows\System32>go version
go version go1.19 windows/amd64

C:\Windows\System32>go get github.com/gotk3/gotk3/gtk
go: go.mod file not found in current directory or any parent directory.
        'go get' is no longer supported outside a module.
        To build and install a command, use 'go install' with a version,
        like 'go install example.com/cmd@latest'
        For more information, see https://golang.org/doc/go-get-install-deprecation
        or run 'go help get' or 'go help install'.

C:\Windows\System32>go install github.com/gotk3/gotk3/gtk@latest
package github.com/gotk3/gotk3/gtk is not a main package

**Comment by BobSynfig** [comment]
github.com/gotk3/gotk3<b>/gtk</B> is only used inside the .go file
```
import (
	"github.com/gotk3/gotk3/gtk"
)
```



Please try again with
`go install github.com/gotk3/gotk3` or `go get github.com/gotk3/gotk3` for _v0.6.1_
or
`go install github.com/gotk3/gotk3@master` or `go get github.com/gotk3/gotk3@master` for _v0.6.2-0.20220628045702-d885f48a5492_

**Comment by DThiebaud** [comment]
Trying that, I get:
dick@dick1:~/src$ go install github.com/gotk3/gotk3@v0.6.1
go install: github.com/gotk3/gotk3@v0.6.1: module github.com/gotk3/gotk3@v0.6.1 found, but does not contain package github.com/gotk3/gotk3
dick@dick1:~/src$ go get github.com/gotk3/gotk3@v0.6.1
go get: package github.com/gotk3/gotk3 is in the main module, so can't request version v0.6.1
dick@dick1:~/src$ go get github.com/gotk3/gotk3@master
go get: package github.com/gotk3/gotk3 is in the main module, so can't request version master
dick@dick1:~/src$ go install github.com/gotk3/gotk3@master
go install: github.com/gotk3/gotk3@master: module github.com/gotk3/gotk3@master found (v0.6.2-0.20220628045702-d885f48a5492), but does not contain package github.com/gotk3/gotk3
dick@dick1:~/src$

**Comment by BobSynfig** [comment]
The last error is ok, you should focus on v0.6.1.
Just retry without adding the version
`go get github.com/gotk3/gotk3`
It seems it is already referenced in your main.go

You can also check the content of go.mod and go.sumto see if the package has been installed.
Also avoid mix of versions in these files (0.6.1 + 0.6.2)

_go: go.mod file not found in current directory or any parent directory._
Of course, if these 2 files don't exist, you need first to do a _`go init mod yourModuleName`_

_Note: When you will try to do your first build, it will take a long time looking like frozen but after it is very fast.
Compiling the first example took something like 10 minutes on my machine!_

**Comment by DThiebaud** [comment]
I didn't realize I needed to create a go.mod file--I hadn't used go in several years. Once I did, I worked as it should.

go get github.com/gotk3/gotgk3 i

installs v0.6.1 successfully.

go get github.com/gotk3/gotk3@master

installs v0.6.2 successfully, upgrading v0.6.1

By the way there is a typo in your last answer. It says "go init mod ModuleName" but it should be "go mod init ModuleName"

BobSynfig, thank you very much for your help.
